Sprint-3/reading-list/script.js

```diff
@@ -21,3 +22,30 @@ const books = [
   },
 ];
 
+function readingList(books) {
+  const list = document.getElementById("reading-list");
+
+  books.forEach((book) => {
+    const listItem = document.createElement("li");
+    listItem.className = "book-item";
+    listItem.style.backgroundColor = book.alreadyRead ? "#c8f7c5" : "#f7c5c5";
```

Comment:
I like the way you used a ternary operator here to determine the background color. 👍
However, it's generally preferred to keep CSS styling within your CSS styling files. This will make it easier to maintain styling as your application grows. Can you make this change?

https://www.w3schools.com/html/html_classes.asp -> look at the _Multiple Classes_ section